src/SFA.DAS.EmployerFinance.Api/appsettings.Development.json

```diff
@@ -5,5 +5,8 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "ApplicationInsights": {
```

Comment:
This isn't needed as the key will be set by the `APPINSIGHTS_INSTRUMENTATIONKEY` environment variable that devops provide.